pastis/io/write/hiclib.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+
+
+def write_counts(X, filename, lengths=None, resolution=50000, copy=True):
```

Comment:
Why has this changed module? While I do not use this function, this is a backward incompatible change. If we are to move it, we should do a proper deprecation cycle (in which case I would remove the function from the package)

Reply:
Good catch, thanks for pointing out the backward incompatibility. I'll update the io submodule so that it is no longer backward incompatible. I'm ambivalent about eventually deprecating this function, so whatever you think is best